db/migrate/20150521145620_add_fulltext_indexes_on_repositories_namespaces.rb

```diff
@@ -1,6 +1,12 @@
 class AddFulltextIndexesOnRepositoriesNamespaces < ActiveRecord::Migration
   def change
-    add_index :namespaces, :name, type: :fulltext, name: 'fulltext_index_namespaces_on_name'
-    add_index :repositories, :name, type: :fulltext, name: 'fulltext_index_repositories_on_name'
+    # This `if` statement has been added after this migration was
+    # created. Modifying migrations is generally a *bad* idea but this will only
+    # apply to PostgreSQL deployments that start from scratch, which haven't
+    # been supported until now.
+    if ENV["PORTUS_DB_ADAPTER"].blank? || ENV["PORTUS_DB_ADAPTER"] == "mysql"
```

Comment:
Shouldn't it be `postgres` instead of `mysql`?

Reply:
Not really. This `if` wants to guarantee that mysql works as usual. So, if it's mysql, let's add this index (that it will be removed by the other migration file)

Reply:
Do we need the `if` overall? I'm confused. 😕

Reply:
We do because this type is not supported by PostgreSQL (or it comes with a different name, we have to investigate more on this with #1459). So, in PostgreSQL this migration fails.

Reply:
Now it makes sense. 😄